Volume/View Models/BigReadArticleRow.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  BigReadArticleRow.swift
+//  Volume
+//
+//  Created by Cameron Russell on 10/30/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct BigReadArticleRow: View {
+    let article: Article
+    
+    var body: some View {
+        VStack(spacing: 15) {
+            if article.image != nil {
+                Image(article.image!)
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .frame(width: 180, height: 180)
+                    .clipped()
+                    .padding(.bottom, 4)
+            } else {
+                Rectangle() // TODO: Custom image displaying beginning of article in large font
+                    .frame(width: 180, height: 180)
+                    .foregroundColor(.blue)
+            }
+            ArticleInfo(article: article)
+        }
+        .frame(width: 180, height: 300)
```

Comment:
What element is this? I can't find on zeplin what has a width of 180 and height of 300.

Reply:
looks like the larger vertical-style "cells" on the home page?

Reply:
It's technically 321 on Zeplin but it fits in 300. And yeah, this is the frame for 1 entire Big Read article preview